incubator/spot-recommender/README.md

```diff
@@ -0,0 +1,72 @@
+# Hollowtrees Spot Recommender
+
+[SpotRecommender]https://github.com/banzaicloud/spot-recommender/blob/master/README.md) Spot instance recommender is a building block of the Hollowtrees project. 
```

Comment:
Are you sure this one is the right link, not this one? https://github.com/banzaicloud/spot-recommender